include/openmc/material.h

```diff
@@ -151,12 +155,22 @@ class Material {
   //! \return Temperature in [K]
   double temperature() const;
 
+#ifdef NCRYSTAL
+  //! Gwet pointer to NCrystal material object
+  //! \return Pointer to NCrystal material object
+  std::shared_ptr<const NCrystal::ProcImpl::Process> m_NCrystal_mat() const {return m_NCrystal_mat_; };
+#endif
+
   //----------------------------------------------------------------------------
   // Data
   int32_t id_ {C_NONE};                 //!< Unique ID
   std::string name_;                    //!< Name of material
   vector<int> nuclide_;                 //!< Indices in nuclides vector
   vector<int> element_;                 //!< Indices in elements vector
+#ifdef NCRYSTAL
+  std::string cfg_; //!< NCrystal configuration string
+  std::shared_ptr<const NCrystal::ProcImpl::Process> m_NCrystal_mat_;
```

Comment:
See [here](https://docs.openmc.org/en/latest/devguide/styleguide.html#naming) for rules on naming. For member variables, we always use lower-case with underscores. I would also add a "ncrystal" prefix for the `cfg_` variable:
```suggestion
  std::string ncrystal_cfg_; //!< NCrystal configuration string
  std::shared_ptr<const NCrystal::ProcImpl::Process> ncrystal_mat_;
```